src/components/panels/StatusPanel.vue

```diff
@@ -88,7 +88,7 @@
                 </v-container>
                 <v-divider class="mt-0 mb-0"></v-divider>
             </template>
-            <v-tabs v-model="activeTab" fixed-tabs dark>
+            <v-tabs v-model="activeTab" fixed-tabs>
```

Comment:
I think this will be wrong, if you just delete the dark option with no condition.